gradle/dependency-versions.gradle

```diff
@@ -25,6 +25,7 @@
   commonsCollectionVersion = "3.2.1"
   commonsHttpClientVersion = "3.1"
   commonsLang3Version = "3.4"
+  cytodynamicsVersion = "0.1.1-SNAPSHOT"
```

Comment:
Leaving a reminder: This needs to be replaced by a real version of cytodynamics before checking this in.

Reply:
I think you'll also need to add it to the LICENSE file.

Reply:
We don't directly bundle anything from cytodynamics (BSD2 license) inside samza. We just depend on it. Do we need to add anything? I looked at the LICENSE file, and it seems to just include certain bundled components (I think for the website?). However, it doesn't seem to have anything about libraries (e.g. kafka, apache commons, guava, etc.).